.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git/
+build/
+coverage/
+node_modules/
```

Comment:
This file makes building the Docker image locally faster by not sending unnecessary files (_build context_) to the Docker daemon.